### PR TITLE
chore: make contract crates compilable

### PR DESCRIPTION
**Pull request by enitrat**
## Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [x] Build-related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Resolves: #

## What is the new behavior?

- Allows compiling the contract crate to output a compiled contract that can be deployed.
- 
-

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by enitrat** [comment]
@Eikix why does `compute_starknet_address` fail?

File: crates/eoa/Scarb.toml [hunk 1]
```diff
@@ -7,3 +7,6 @@ version = "0.1.0"
 [dependencies]
 starknet.workspace = true
 evm = { path = "../evm" }
+
+[tool]
```
Comment (Eikix) [inline crates/eoa/Scarb.toml:11]: does this make our pre-commit hook finally work?

**Comment by Eikix** [comment]
> @Eikix why does `compute_starknet_address` fail?
> 
> Is it possible that the class hash has changed with update to 2.3?

Weird, I think Harsh's PR has a similar problem. I don't understand why

**Comment by Eikix** [comment]
Weirdly enough I rebased the feat/coinbase PR on main, and ran scarb test, but it passes
What could be different here?

**Comment by enitrat** [comment]
The only change is import ordering which shouldn't modify the class hash of EOA 🤔

File: crates/eoa/Scarb.toml (hunk 1 above)
Comment (enitrat) [reply crates/eoa/Scarb.toml:11]: no, this just makes all crates use import formatting

**Review by Eikix — APPROVED** [review]
lgtm

**Pull request merged**